src/bp/core/services/ghost/disk-driver.ts

```diff
@@ -48,6 +48,14 @@ export default class DiskStorageDriver implements StorageDriver {
     }
   }
 
+  async moveFile(fromPath: string, toPath: string): Promise<void> {
```

Comment:
We can use the @WrapWithError extension instead of try catches now :-)

Reply:
Good point, I forgot about that